### PR TITLE
Add gov proposal handler for acl

### PR DESCRIPTION
**Pull request by BrandonWeng**
Adding a handler for handling gov proposals from CLI in order to update the dependency mappings for access control module. Will follow up with another PR to handle deletes

### Testing 
Ran these commands and see that the dep mapping updated
```
seid tx staking delegate seivaloper1r9syjw4kdpjr8a6v38qmvn4dmmrnl869kmgsvr 59999000001341816640usei --from brando --chain-id sei-chain -b block -y --fees 2000usei

seid tx gov submit-proposal update-resource-dependency-mapping ../brando/acl-gov.json --from brando  --fees=10000000usei --gas=5000000  --chain-id sei-chain  -b block

seid tx gov deposit 1 10000000usei --chain-id sei-chain --from brando --fees=10000000usei --gas=5000000 -b block -y

seid tx gov vote 1 yes --chain-id sei-chain --from brando -b block -y --fees=10000000usei
``` 

image

### CODE REVIEW

File: x/accesscontrol/examples/update-access-dep-mapping-proposal.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (philipsu522) [inline x/accesscontrol/examples/update-access-dep-mapping-proposal.json:1]: nice

File: x/accesscontrol/handler.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+
+	"github.com/cosmos/cosmos-sdk/x/accesscontrol/keeper"
+	"github.com/cosmos/cosmos-sdk/x/accesscontrol/types"
+)
+
+func HandleMsgUpdateResourceDependencyMappingProposal(ctx sdk.Context, k *keeper.Keeper, p *types.MsgUpdateResourceDependencyMappingProposal) error {
+	for _, resourceDepMapping := range p.MessageDependencyMapping {
+		k.SetResourceDependencyMapping(ctx, resourceDepMapping)
```
Comment (philipsu522) [inline x/accesscontrol/handler.go:14]: do we do any checks over `resourceDepMapping`?

**Review by philipsu522 — APPROVED** [review]

File: x/accesscontrol/client/cli/tx.go [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+	return cmd
+}
+
+func MsgUpdateResourceDependencyMappingProposalCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update-resource-dependency-mapping [proposal-file]",
+		Args:  cobra.ExactArgs(1),
+		Short: "Submit an UpdateResourceDependencyMapping proposal",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			println("MsgUpdateResourceDependencyMappingProposalCmd:HANDLING CLI REQUEST")
```
[the remark this answers is not in the document]
Comment (BrandonWeng) [reply x/accesscontrol/client/cli/tx.go:43]: oh shot, thanks for the catch, i was looking for log statements 😵‍💫

File: x/accesscontrol/handler.go (hunk 2 above)
Comment (BrandonWeng) [reply x/accesscontrol/handler.go:14]: Mainly just the proto serialization checks -- all the fields that are not free form are ENUMs so the proto serialization should also cover those cases

**Pull request merged**